src/txn/txn_timestamp.c

```diff
@@ -74,7 +74,7 @@ __wt_txn_parse_timestamp(
 static void
 __txn_get_read_timestamp(WT_TXN_SHARED *txn_shared, wt_timestamp_t *read_timestampp)
 {
-    WT_ORDERED_READ(*read_timestampp, txn_shared->read_timestamp);
+    WT_READ_ONCE(*read_timestampp, txn_shared->read_timestamp);
```

Comment:
I'd be happy to go back on this conversion until a more thorough review of timestamp ordering is undertaken but as far as I can tell these read barriers are trying to do READ_ONCE.